packages/react-async/src/useAsync.js

```diff
@@ -154,33 +165,44 @@ const useAsync = (arg1, arg2) => {
   )
 }
 
+export class FetchError extends Error {
```

Comment:
Just as a preparation for TypeScript, we should set the prototype manually here.
https://github.com/Microsoft/TypeScript-wiki/blob/master/Breaking-Changes.md#extending-built-ins-like-error-array-and-map-may-no-longer-work

Reply:
👍